src/tm_mad/fs_lvm/delete

```diff
@@ -44,37 +44,55 @@ source ${DRIVER_PATH}/../../datastore/libfs.sh
 DST_PATH=`arg_path $DST`
 DST_HOST=`arg_host $DST`
 
-DS_SYS_ID=$(echo $DST_PATH | grep -E '\/disk\.[[:digit:]]+$' | $AWK -F '/' '{print $(NF-2)}')
+if [ "${DST_PATH##*/}" = "$VM_ID" ]; then
+    DS_SYS_ID=$(echo $DST_PATH | $AWK -F '/' '{print $(NF-1)}')
+else
+    DS_SYS_ID=$(echo $DST_PATH | grep -E '\/disk\.[[:digit:]]+$' | $AWK -F '/' '{print $(NF-2)}')
```

Comment:
I think this else clause is not needed, using empty "DS_SYS_ID" later (line 99) it detects if it's going to delete LV or just directory. Then also the changes in the DELETE_CMD are not required.

Reply:
We need "DS_SYS_ID" in both cases:
1. If we removing disk, we should go on the BRIDGE_LIST node and remove the disk.
2. If we removing directory, we should go on the BRIDGE_LIST node and remove the directory.

If we would not do that, it will try to remove directory on the controller, what cause an error if there is no mounted datastore

Reply:
OK, clear, then could you also remove the `if` statement on line 99 + it's else 125..127?

Reply:
ok, done